app/src/main/java/de/danoeh/antennapod/ui/home/settingsdialog/ItemMoveHandler.java

```diff
@@ -0,0 +1,5 @@
+package de.danoeh.antennapod.ui.home.settingsdialog;
+
+public interface ItemMoveHandler {
```

Comment:
I tried it but unfortunately, `BiConsumer` doesn't seem to be available for now. 
The regular `Consumer` and `BiConsumer` from `java.util.function` are not possible to use to support Android < 24. For `Consumer`, you can instead import it from `androidx.core.util` to get it to work. Unfortunately, BiConsumer doesn't seem to be available in that package. It seems it can be used if you turn on desugaring for the project but I didn't want to do that in this PR.

Or do you have other suggestions here?

Reply:
Could create a class with the two positions and use a regular `Consumer` that returns and object of that class as well if you think that is better.